src/app/components/team/Similarity/SimilarityComponent.js

```diff
@@ -269,7 +220,60 @@ const SimilarityComponent = ({
               labelPlacement="end"
               onChange={() => handleSettingsChange('single_language_fact_checks_enabled', !settings.single_language_fact_checks_enabled)}
             />
-          </Box>
+          </div>
+          <div className={inputStyles['form-fieldset']}>
+            <SwitchComponent
+              checked={settings.master_similarity_enabled}
+              className={inputStyles['form-fieldset-field']}
+              label={
+                settings.master_similarity_enabled ?
+                  <FormattedMessage defaultMessage="Automated matching is ON" description="Switch input label when switch is set to automatic matching" id="similarityComponent.masterSwitchLabelOn" /> :
+                  <FormattedMessage defaultMessage="Automated matching is OFF" description="Switch input label when switch is set to not automatic match" id="similarityComponent.masterSwitchLabelOff" />
+              }
+              labelPlacement="end"
+              onChange={() => handleSettingsChange('master_similarity_enabled', !settings.master_similarity_enabled)}
+            />
+            { settings.master_similarity_enabled &&
+              <>
+                <br />
+                <SwitchComponent
+                  checked={settings.date_similarity_threshold_enabled}
+                  className={inputStyles['form-fieldset-field']}
+                  label={
+                    <FormattedMessage
+                      defaultMessage="Limit matching to recently submitted content"
+                      description="Allow user to enable/disable similarity matching based on submitted date"
+                      id="similarityComponent.dateThresholdSettings"
+                    />
+                  }
+                  labelPlacement="end"
+                  onChange={() => handleSettingsChange('date_similarity_threshold_enabled', !settings.date_similarity_threshold_enabled)}
+                />
+                { settings.date_similarity_threshold_enabled &&
+                  <FormattedMessage
+                    defaultMessage="Only similar content last submitted more than {maxTime} {count, plural, one {month} other {months}} ago will be suggested."
```

Comment:
Oh, I now see, looking at the previous copy:

![image](https://github.com/user-attachments/assets/e71ba2a8-1170-4a3d-9543-378fe1ee7e03)

I remember this date threshold is to make older content "less relevant" so to say. This means that content older than `maxTime` will not be automatically be a confirmed match, but a suggestion, instead. I had on my mind the other way, when I read the new copy thinking that only more recent (hence less then) would be suggested.

Still, I think the new phrasing changes the meaning a little bit: If we say `"Only similar content last submitted more than {maxTime} {count, plural, one {month} other {months}} ago will be suggested."` implies that older content won't be suggested, but that's not really the case. Older content will still be suggested but will not be a confirmed matched without human review. Is my understanding correct @caiosba ?

Reply:
I think this is exactly why I was changing the language a bit @amoedoamorim ...because I was confused so I was trying to make it clearer for myself, but you are right I might have changed the meaning.
Lets see what @caiosba says what the reality is, then we can choose the right phrasing, I was confused by the original text

Reply:
> Older content will still be suggested but will not be a confirmed matched without human review

That's correct, @amoedoamorim

Reply:
Oh I see, I think what was causing my confusion was the word "only". I changed the language back to mostly what it was before with just a bit more clarity:

### OLD
`Similar content last submitted more than {maxTime} months ago will only be suggested`

### NEW
`Similar content last submitted more than {{count}} months ago will only be suggested, not automatically confirmed as a matched.`

I think this is accurate now @amoedoamorim 
CC @caiosba

Reply:
:lgtm: